assets/sass/application.scss

```diff
@@ -450,6 +450,12 @@ input[type="time"] {
   background-color: #FFF0D5;
 }
 
+.mandatory-warning-container {
+  @extend %container-shared;
+  background-color: #ffffff;
+  padding-left: 0%;
+}
+
```

Comment:
Reluctant to keep in the `application.scss` file but decided to keep here to maintain consistency as the other container classes are maintained here